PrioritiesSprint3.md

```diff
@@ -0,0 +1,67 @@
+These are the priorities for Sprint number 3, which runs from April 29, 2024 to May 26, 2024. 
+In this sprint, the Eel build is in CI and the Octopus build is in Test. Each priority has a list of subtasks that are to be completed within this sprint.
+
+Each priority has a list of subtasks that are to be completed within this sprint.  
+
+Note that ARAs will be required to implement either MCQ or Pathfinder (or both).
+
+# Develop Pathfinder Query
+## Pre-development work:
+- Design new landing page to distinguish types of interactions (templated query, pathfinder, multiquery(?)) (UI)
+- Design query selection interface - Dependencies: decisions on categories of nodes to be allowed as starting points and any other factors allowed for query selection (UI, ARAs)
+- User testing path grouping strategies (UI, ARAs)
+- Design results page path  and graph display (similar to results now) - Dependencies: understanding of the number of paths and how they will be grouped (UI, ARAs)
+- Research to identify new facets needed. e.g. facets for other categories - Dependencies: pathfinder data examples (UI, ARAs)
+- User research filter designs for new filters like metaedge type or other. (UI)
+- Provides example datasets to UI (ARAs)
```

Comment:
What sort of dataset is being requested here?

Reply:
@uhbrar ?

Reply:
The "dataset" is just the outputs for the 4 example queries.

Reply:
So do we want everything  ARA to provide outputs for all 4 example queries?  Or an ARA or what?